docs/sources/api-response-examples/atlassian/jira/sanitized/issues_by_jql.json

```diff
@@ -44,7 +44,7 @@
           "accountType":"atlassian",
           "active":true
         },
-        "timespent":null,
+        "timespent":0,
```

Comment:
Fixed type; it is an integer and it was set as string. But now, is converted from null to zero when no value...